### ISSUE TITLE
UserWarning during usage of head() and tail() functions

### ISSUE THREAD

**Issue by shafeneger** [ibmdbanalytics/ibmdbpy#16]
When the `head()` / `tail()` function is used on an `Ida.DataFrame` and no `indexer` is set, then the user gets the following `UserWarning`:
```
UserWarning: Row order is not guaranteed if no indexer was given and the dataset was not sorted.
  warnings.warn("Row order is not guaranteed if no indexer was given and the dataset was not sorted.")
```

Is this warning really necessary, is it possible to remove the warning ? I mean during creation of the `Ida.DataFrame`, the default is `indexer=None`. It is a bit overkill to see it whenever I use `head()/tail()`.

**Comment by nicoleschoen** [comment]
The rationale behind that is that the indexer column is also used by machine learning algorithms and there you should explicitly choose the indexer column. But I agree that in case of head and tail this is not the case, we could just always sort instead of showing the warning, e.g. by the first column.
Will change that.

**Comment by nicoleschoen** [comment]
The change is implemented, find it in ibmdbpy (0.1.4). 
We now use by default the first numeric column and if no numeric column is available we use the first column. I also fixed the index output for dataframes with an indexer.
